app/models/user.rb

```diff
@@ -377,7 +375,7 @@ def identity_verified?
   end
 
   def identity_verified_with_facial_match?
-    FACIAL_MATCH_IDV_LEVELS.include?(active_profile&.idv_level)
+    identity_verified? && active_profile.facial_match?
```

Comment:
I think that this would be a more direct check?
```suggestion
    active_profile&.facial_match?
```

Reply:
That was my 1st approach. Two tests failed because they were asserting `false` instead of `falsey`. Instead of fixing those tests, I thought it would be just as easy to be explicit.

Reply:
in that case, might be clearer as this, since it's not immediately obvious that  `identity_verified?` doesn't do additional checks
```suggestion
    active_profile.present? && active_profile.facial_match?
```

Reply:
I was debating between `identity_verified?` and `active_profile.present?` myself. Happy to get the 2nd opinion.